src/autoscaler/api/config/config.go

```diff
@@ -52,80 +53,80 @@ var defaultLoggingConfig = helpers.LoggingConfig{
 }
 
 type SchedulerConfig struct {
-	SchedulerURL   string          `yaml:"scheduler_url"`
-	TLSClientCerts models.TLSCerts `yaml:"tls"`
+	SchedulerURL   string          `yaml:"scheduler_url" json:"scheduler_url"`
```

Comment:
Q. Why we want to support json here ? Is this because, mta deployments support json and yaml variants ?

Reply:
So the config now gets read from a user provided service, which is in json format as opposed to the current yaml from file format.